src/Aspire.Hosting/Kafka/KafkaBuilderExtensions.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Sockets;
+using System.Text.Json;
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting;
+
+public static class KafkaBuilderExtensions
+{
+    /// <summary>
+    /// Adds a Kafka broker container to the application.
+    /// </summary>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{KafkaContainerResource}"/></returns>
+    public static IResourceBuilder<KafkaContainerResource> AddKafkaBrokerContainer(this IDistributedApplicationBuilder builder, string name)
+    {
+        const int kafkaBrokerPort = 9092;
```

Comment:
Should this be optionally configurable?
Does anyone change the default port anyway?

Reply:
Host port is now configurable.